en/micro-integrator/docs/references/connectors/amazonsqs-connector/amazonsqs-inbound-endpoint-example.md

```diff
@@ -0,0 +1,106 @@
+# AmazonSQS Inbound Endpoint Example
+
+The AmazonSQS inbound endpoint allows you to consume messages via WSO2 EI. The inbound endpoint acts as a message consumer. It creates a connection to an Amazon SQS account, consumes messages from the Amazon SQS queue, and injects the messages into the WSO2 EI sequence.
```

Comment:
`AmazonSQS Inbound Endpoint` shall we make this Bold? 
The **AmazonSQS Inbound Endpoint** allows you to connect to Amazon and consume messages form an Amazon SQS queue. The messages are then injected into a WSO2 EI mediation engine for further processing and mediation.

Reply:
Updated.